scripts/manage/restore_backup_to_cloud_shell.sh

```diff
@@ -117,5 +117,6 @@ rm -rf $TEMP_DIR
 
 bold "Configuration applied. To diff this config with what was last deployed, go to:"
 bold "https://source.cloud.google.com/$PROJECT_ID/$EXISTING_CSR_REPO/+/$GIT_HASH...master"
+bold "Note: Any pre-existing configuration for secure access to Spinnaker via IAP has not been modified."
```

Comment:
[minor] "has been left unchanged" or so? Also, "pre-existing" may be taken as meaning "before you created Spinnaker for GCP", which is obviously not the case.

Just to clarify: what we mean here is that switching to a different config leaves the setup for secure external access unchanged. Is it fair to expect users to understand what that means? I assume they don't know *which* bits of config are actually affected, and can thus also not judge if this is a potential security problem, or not?

Is there something we can say to indicate e.g. that this might leave "dangling resources" around, but should not put you in an insecure state?

Reply:
I've wordsmithed a few more options to indicate the IAP config still exists but remains secure. Thoughts? Also @dorbin could you take a look as well?

`Note: Any previously created configuration for secure access to Spinnaker via IAP has been left unchanged.`

--- OR ---

`Note: If Spinnaker was previously configured for secure access via IAP, that configuration has been left unchanged. Spinnaker may still be accessible via the previously configured secure endpoint.`

Reply:
"Note: If secure access via IAP is already configured, that configuration has not been changed. Users can still access Spinnaker using the previously configured endpoint."

Reply:
Sounds good, thoughts on above @demobox ?

Reply:
@dorbin I suspect that isn't quite accurate since *some* parts of the configuration *have* in fact changed (e.g. Spinnaker is no longer checking to see if that particular user has permissions).

The gist of the matter here is that, as a user, you may be thinking that reverting to a pre-IAP configuration cleans up IAP, but that is not in fact the case, although it *does* change your configuration. The IAP part of the configuration remains; what we want to reassure you of is that that is not causing a security vulnerability (even though Spinnaker remains publicly accessible over the internet).

I wonder if we should add a short description (or script?) somewhere that describes what you need to do to remove the IAP config?

Reply:
For your last sentence, @demobox , I leave it to you and @louisjimenez . 

Another attempt at that sentence, though: 
"If secure access via IAP is already configured, that access is left unchanged and remains secure."

Reply:
@demobox My understanding was that the IAP config did not change and was still accessible through the endpoint. I do see how other items such as the spin config will change. I tested out reverting and found that both port forwarding and IAP access continued working. 

Do you think there is still value in merging the revised sentence above or do you think that steps for removing IAP are what users are looking for here?

Reply:
> If secure access via IAP is already configured, that access is left unchanged and remains secure.

That works for me - thanks for the suggestions, @dorbin!